tests/resources/reporting_round.csv

```diff
@@ -4,10 +4,12 @@ bb265da3-999e-4282-aa2c-0fa85da503a1,2,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2022
 de52f699-f26f-44ff-b7ca-5cc834300500,3,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2022-10-01 00:00:00,2023-03-31 23:59:59,,
 eede818b-8697-4034-a4a2-7c35dff080bb,4,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2023-04-01 00:00:00,2023-09-30 23:59:59,,2023-12-04 23:59:59
 4428f656-5997-42c2-9149-b2303a6f1765,5,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2023-10-01 00:00:00,2024-03-31 23:59:59,,2024-05-28 23:59:59
+4428f654-5997-42c2-9149-b2303a6f1765,6,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2024-04-01 00:00:00,2024-09-30 23:59:59,,

```

Comment:
This need to be new random uuid4s, not copies (try `flask db-data seed-ref` to test that it ingests correctly). Also - can we get reporting period start+end populated too?

Reply:
As mentioned in the meeting one-character tweaks were made after copying but yeah it's not obvious at all at a glance haha! @israr-ulhaq would you mind looking at this one?